specification/maps/data-plane/Timezone/preview/1.0/timezone.json

```diff
@@ -330,7 +423,7 @@
     "/timezone/windowsToIana/{format}": {
       "get": {
         "description": "__Windows to IANA Time Zone__\n\n\n**Applies to**: S0 and S1 pricing tiers.\n\n\nThis API returns a corresponding IANA ID, given a valid Windows Time Zone ID. Multiple IANA IDs may be returned for a single Windows ID. It is possible to narrow these results by adding an optional territory parameter.",
-        "operationId": "GetTimezoneWindowsToIANA",
+        "operationId": "GetIANAMetadata",
```

Comment:
I don't think this returns metadata... It transforms windows id to one or more iana ids. Any better name?

Reply:
Yes this name is wrong. ConvertWindowsTimezoneToIANA